journal-letters/modernized/01-george-fox-testimony.adoc

```diff
@@ -1,4 +1,4 @@
-== George Fox`'s testimony concerning William Caton.
+== _George Fox`'s testimony concerning_ William Caton
```

Comment:
Yup, you've got it exactly how the original is formatted, but I think I lean towards _removing_ the italics here, since I feel like they're more style than data, so let's nix the italics. 👍